netmiko/endace/endace_ssh.py

```diff
@@ -1,18 +1,36 @@
+from typing import Any
+
 from netmiko.cisco_base_connection import CiscoSSHConnection
 import re
 
 
 class EndaceSSH(CiscoSSHConnection):
-    def disable_paging(self, command="no cli session paging enable", delay_factor=1):
+    def disable_paging(
+        self,
+        command: str = "no cli session paging enable",
+        delay_factor: float = 1.0,
+        *args: Any,
+        **kwargs: Any
+    ) -> str:
         return super().disable_paging(command=command, delay_factor=delay_factor)
 
-    def enable(self, cmd="enable", pattern="", re_flags=re.IGNORECASE):
+    def enable(
+        self, cmd: str = "enable", pattern: str = "", re_flags: int = re.IGNORECASE
+    ) -> str:
         return super().enable(cmd=cmd, pattern=pattern, re_flags=re_flags)
 
-    def check_config_mode(self, check_string="(config) #"):
+    def check_config_mode(
+        self, check_string: str = "(config) #", *args: Any, **kwargs: Any
+    ) -> bool:
         return super().check_config_mode(check_string=check_string)
 
-    def config_mode(self, config_command="conf t", pattern=""):
+    def config_mode(
+        self,
+        config_command: str = "conf t",
+        pattern: str = "",
+        *args: Any,
+        **kwargs: Any

```

Comment:
Probably just add the one additional argument `re_flags: int = 0` (with the standard default value instead of *args/**kwargs). This argument will not be used.